packages/okutrade/src/utils.ts

```diff
@@ -56,6 +88,15 @@ const chainToWETH: Record<number, Address> = {
   [Chains.BASE]: '0x4200000000000000000000000000000000000006',
 }
 
+const chainIdToName: Record<number, string> = {
+  1: 'ethereum',
```

Comment:
one small nit, but could you use the Chains enum here like used in `chainToWeth`

Reply:
done